src/fidesops/ops/service/email/email_dispatch_service.py

```diff
@@ -54,10 +55,15 @@ def _build_email(
     body_params: Union[SubjectIdentityVerificationBodyParams],
 ) -> EmailForActionType:
     if action_type == EmailActionType.SUBJECT_IDENTITY_VERIFICATION:
+        template = get_email_template(action_type)
         return EmailForActionType(
             subject="Your one-time code",
-            # for 1st iteration, below will be replaced with actual template files
-            body=f"<html>Your one-time code is {body_params.access_code}. Hurry! It expires in 10 minutes.</html>",
+            body=template.render(
+                {
+                    "code": body_params.verification_code,
+                    "minutes": body_params.get_verification_code_ttl_minutes(),
```

Comment:
I think that was done because each template may have slightly different parameters the the template but they would all get passed on in `body_params`. It could definitely lead to errors if the function isn't set up properly as more emails are added.

Maybe we could do a refactor later if it proves to be not very maintainable?